### PR TITLE
Add WorkerInfoField test in CapacityCommandTest

### PR DESCRIPTION
**Pull request by YichuanSun**
### What changes are proposed in this pull request?

As the title. If Options has a WorkerInfoField which POptions does not have, or if POptions has a WorkerInfoField which Options does not have, this test will be failed. The latest test version can not check these two error, as shown in the PR below.

### Why are the changes needed?

[Add missing variant in gRPC WorkerInfoField](https://github.com/Alluxio/alluxio/pull/16457)

### Does this PR introduce any user facing changes?

No.

### CODE REVIEW

**Comment by YichuanSun** [comment]
@dbw9580 PTAL.

File: shell/src/test/java/alluxio/cli/fsadmin/report/CapacityCommandTest.java [hunk 1]
```diff
@@ -153,6 +154,19 @@ public void longWorkerNameCapacity() throws IOException {
     }
   }
 
+  /**
+   * Check whether WorkerInfoField class and WorkerInfoField in proto file has identical fields.
+   */
+  @Test
+  public void workerInfoFieldCapacity() throws IOException {
+    // If Options has a WorkerInfoField which POptions does not have, throw IOException.
+    GetWorkerReportOptions.defaults().toProto();
```
Comment (dbw9580) [inline shell/src/test/java/alluxio/cli/fsadmin/report/CapacityCommandTest.java:163]: for a test to work, you need to encode the property being tested as an assertion, e.g.

```java
assertEquals(GetWorkerReportPOptions.getDefaultInstance(),
    GetWorkerReportOptions.defaults().toProto());
```

Ideally, you don't need to waste time writing comments explaining what is being tested, but rather the test code should be self-explanatory as if you are reading a plain English sentence. In this case, you can read it aloud as "the default value of a gRPC `GetWorkerReportOptions` equals the default value of the java instance returned by the `default` method and then converted to the gRPC representation."

File: shell/src/test/java/alluxio/cli/fsadmin/report/CapacityCommandTest.java [hunk 2]
```diff
@@ -153,6 +154,19 @@ public void longWorkerNameCapacity() throws IOException {
     }
   }
 
+  /**
+   * Check whether WorkerInfoField class and WorkerInfoField in proto file has identical fields.
+   */
+  @Test
+  public void workerInfoFieldCapacity() throws IOException {
```
Comment (dbw9580) [inline shell/src/test/java/alluxio/cli/fsadmin/report/CapacityCommandTest.java:161]: give the test method a better name that directly reflects what is being tested.

**Review by dbw9580 — CHANGES_REQUESTED** [review]
On second thought, this should belong in its own unit test class. please create a new UT for `GetWorkerReportOptions` and add the test there.

**Comment by dbw9580** [comment]
@YichuanSun can you please address the review comments?

File: core/client/fs/src/test/java/alluxio/client/block/options/GetWorkerReportOptionsTest.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class GetWorkerReportOptionsTest {
+  /**
+   * Check whether WorkerInfoField class and WorkerInfoField in proto file has identical fields.
+   */
+  @Test
+  public void GetWorkerReportOptionsMapTest() {
```
Comment (dbw9580) [inline core/client/fs/src/test/java/alluxio/client/block/options/GetWorkerReportOptionsTest.java:25]:
```suggestion
  public void identicalFieldsForWorkerInfoAndRange() {
```

**Review by dbw9580 — APPROVED** [review]
LGTM

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Pull request merged**